package.json

```diff
@@ -77,8 +66,11 @@
   "license": "MIT",
   "devDependencies": {
     "@netlify/eslint-config-node": "^5.1.6",
-    "ava": "^4.0.0",
+    "ava": "https://github.com/avajs/ava",
```

Comment:
Support for sorting tests files was merged to AVA's default branch, but not released yet officially.

Reply:
Note: we might want to wait until the next release to merge this PR then, to keep using regular version numbers in `devDependencies`.